core/types/authorization.go

```diff
@@ -20,39 +20,27 @@ import (
 type Authorization struct {
 	ChainID *uint256.Int      `json:"chainId"`
 	Address libcommon.Address `json:"address"`
-	Nonce   []uint64          `json:"nonce"`
-	V       uint256.Int       `json:"v"`
+	Nonce   uint64            `json:"nonce"`
+	YParity uint256.Int       `json:"v"`
```

Comment:
The json should also be named "yParity" rather than "v".
v =  27 + yParity
or
v = 2*chainId + 35 + yParity
See [EIP-155](https://eips.ethereum.org/EIPS/eip-155)